ipn/ipnstore/store.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package ipnstore provides utilities to persist state
```

Comment:
Oh, sorry, you did have a package comment here. Just add the missing period at the end of the sentence.

But `ipnstore.StateStore` is pretty stuttery (see https://golang.org/doc/effective_go#package-names and its `bytes.Buffer example).

I think `ipnstate.Store` and `ipnstate.FileStore` would be better names.

Reply:
cool, I did my best to avoid the "stuttery", let me know what you think. I am not sure if I managed to be completely pertinent :thinking: